1725216672953	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1725242658120	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1725242658121	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725242658122	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1725242658124	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1725242658190	Sync.SyncScheduler	DEBUG	Woke from sleep.
1725242658191	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1725242658191	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1725242658191	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1725242660192	Sync.Service	DEBUG	User-Agent: Firefox/129.0.1 (Linux x86_64) FxSync/1.131.0.20240812083845.desktop
1725242660192	Sync.Service	INFO	Starting sync at 2024-09-01 21:04:20 in browser session wkNRj2cFQh2K
1725242660193	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1725242660196	Sync.Status	INFO	Resetting Status.
1725242660196	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1725242660197	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1725242660197	FirefoxAccounts	DEBUG	getOAuthToken enter
1725242660198	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725242660198	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725242660198	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1725242660198	Sync.SyncAuthManager	INFO	Getting sync key
1725242660199	FirefoxAccounts	DEBUG	getOAuthToken enter
1725242660199	FirefoxAccounts	TRACE	getCachedToken returning cached token
1725242660199	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1725242660199	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1725242660199	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1725242660199	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1725242660199	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1725242660201	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725242660201	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1725242660202	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1725242660202	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1725242660203	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1725242660203	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1725242660203	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1725242660203	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1725242660203	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1725242660204	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1725242660207	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/214229003/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725242660207	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1725242660207	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1725242660208	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1725242660208	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1725242660209	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1725242660214	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
